### ISSUE TITLE
3 moderate severity vulnerabilities when installing elm-xref

### ISSUE THREAD

**Issue by lucamug** [zwilias/elm-xref#12]
```
semver-regex  <3.1.3
Severity: moderate
Regular Expression Denial of Service (ReDOS) - https://github.com/advisories/GHSA-44c6-4v22-4mhx
fix available via `npm audit fix --force`
Will install elm-xref@0.3.1, which is a breaking change
node_modules/semver-regex
  semver-sort  <=0.0.4
  Depends on vulnerable versions of semver-regex
  node_modules/semver-sort
    elm-xref  >=0.3.2
    Depends on vulnerable versions of semver-sort
    node_modules/elm-xref

3 moderate severity vulnerabilities
```

**Comment by zwilias** [comment]
Fixed in f4944da4aa1d8b9c55b99c9636d732d3122f9ed3, thanks for the report!

**Comment by lucamug** [comment]
Wow, that was super-fast, thank you! ❤️
